apps/omg_db/test/omg_db/db_test.exs

```diff
@@ -78,6 +78,14 @@ defmodule OMG.DBTest do
     {:ok, ["xyz", "vxyz", "wvxyz"]} = OMG.DB.block_hashes([1, 2, 3], pid)
   end
 
+  test "utxo can be fetched by utxo position", %{db_pid: pid} do
+    index = 123
+    item = {{index, index, index}, %{test: :crypto.strong_rand_bytes(index)}}
```

Comment:
true, but since this is ubiquitous in db tests, I'll leave as is for consistency and self-contained PR. If necessary, let's have a separate issue/PR for this